plugins/techdocs/config.d.ts

```diff
@@ -86,6 +86,12 @@ export interface Config {
                * @visibility secret
                */
               secretAccessKey: string;
+              /**
+               * ARN of role to be assumed
+               * attr: 'roleArn' - accepts a string value
+               * @visibility secret
```

Comment:
Is this necessarily a secret thing? I thought the identifier as such should be ok to have visible

Reply:
it is not - it's fine to have visible